chromium_src/chrome/installer/setup/uninstall.cc

```diff
@@ -0,0 +1,68 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/installer/util/brave_shell_util.h"
```

Comment:
Don't you need to add another dependency on `//brave/installer/util` from the `//chrome/installer/setup:setup` GN target for this?

Reply:
Oops, I missed that. Added (https://github.com/brave/brave-core/pull/9056/files#diff-b59cb8001bdafc216af8342ad837f33188b3673c1bbd13ff04794e3b3afc4988R9)